src/itdelatrisu/opsu/options/OptionsOverlay.java

```diff
@@ -1350,6 +1359,27 @@ private void adjustSlider(int mouseX, int mouseY) {
 		}
 	}
 
+	/**
+	 * Increments or decrements slider value if user presses right or left arrow key.
+	 * @param key the pressed key
+	 */
+	private void adjustSliderWithKey(int key) {
```

Comment:
I think instead of passing in the key, you should pass in something more symbolic, like a delta value.